seaborn/matrix.py

```diff
@@ -304,8 +318,10 @@ def heatmap(data, vmin=None, vmax=None, cmap=None, center=None, robust=False,
     robust : bool, optional
         If True and ``vmin`` or ``vmax`` are absent, the colormap range is
         computed with robust quantiles instead of the extreme values.
-    annot : bool, optional
-        If True, write the data value in each cell.
+    annot : bool, or array, optional
```

Comment:
I'd change this to "or rectangular dataset" (for consistency with `data` above because it can be a DataFrame right?)